src/sentry/migrations/0384_backfill_monitor_env_checkins.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 2.2.28 on 2023-03-14 23:45
+
+from django.db import connection, migrations
+from psycopg2.extras import execute_values
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+BATCH_SIZE = 100
+DEFAULT_ENVIRONMENT_NAME = "production"
+
+UPDATE_QUERY = """
+    UPDATE sentry_monitorcheckin
+    SET monitor_environment_id = data.monitor_environment_id
+    FROM (VALUES %s) AS data (id, monitor_environment_id)
+    WHERE sentry_monitorcheckin.id = data.id"""
+
+
+def backfill_monitor_checkins(apps, schema_editor):
+    MonitorCheckIn = apps.get_model("sentry", "MonitorCheckIn")
+    MonitorEnvironment = apps.get_model("sentry", "MonitorEnvironment")
+
+    queryset = RangeQuerySetWrapperWithProgressBar(
+        MonitorCheckIn.objects.all().values_list(
+            "id",
+            "project_id",
+            "monitor",
+            "monitor_environment",
```

Comment:
do these just pull the `id` of the object here? or the actual object?

Reply:
That will pull the whole `MonitorEnvironment` object. You can use `monitor_environment_id` here to just get the id. Do you actually need all of these fields though? I would've thought you'd only need `id`, `monitor_id` and `monitor_environment_id`

Reply:
Right, I only want to pull the `id` haha. And yeah `project` left in accidentally